src/main/resources/project/ec_setup.pl

```diff
@@ -86,6 +86,12 @@
     description => "Puppet version",
     category    => "Resource Management"
 );
+my %PuppetAgentSetConfig = (
+    label       => "Puppet - AgentSetConfig",
```

Comment:
Good point. I think the existing ConfigureAgent procedure is doing the tasks described in "Configure a Puppet Agent Node" in "http://docs.puppetlabs.com/puppet/3.8/reference/post_install.html#configure-a-puppet-agent-node". So we can name accordingly.
Old Name -> New Name
-------------------------------
ConfigureAgent : 'Configure Puppet Agent Node'
AgentSetConfig: 'Update Puppet Agent Configuration'